index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css" />
+</head>
+<body>
+    <header>
+        <img class="pandalogo" src="/sprint image/Property 1=sm.png" alt="pandamarket logo" onclick="window.location.href='/'">
```

Comment:
💊 제안  
img 태그의 alt 속성은 alternative 라는 의미로, 이미지 파일을 다운로드하는 것에 실패해서 이미지 파일을 보여줄 수 없을 때  
어떤 이미지인지 파악할 수 있게 대신 제공되거나, 스크린리더로 읽혀지는 문자를 의미합니다.  
따라서 **가능한 해당 이미지를 잘 표현할 수 있는 문자열을 넣어주는 것이 좋습니다.**  
위의 경우 "판다마켓"이 적절할 것 같네요~

```suggestion
        <img class="pandalogo" src="/sprint image/Property 1=sm.png" alt="판다마켓">
```

다른 코멘트에서도 언급했지만 페이지 이동의 경우, 특수한 동작이 필요한 것이 아니면 a 태그를 이용해서 구현해주세요~